### PR TITLE
Add github membership

### PR DESCRIPTION
**Pull request by eliecharra**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | yes
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #247 
| ❓ Documentation  | yes

## Description

No acceptance for this one guys as I didn't find a simple way to accept invitation within acceptance test

### CODE REVIEW

**Review by wbeuil — CHANGES_REQUESTED** [review]
Small typo and wonder why you didn't make any acceptance tests ?

**Comment by eliecharra** [comment]
> Small typo and wonder why you didn't make any acceptance tests ?

As I said in the PR description :

> No acceptance for this one guys as I didn't find a simple way to accept invitation within acceptance test

**Comment by wbeuil** [comment]
Sorry forgot what you said in the description of the PR. Then just the small typo and I'm good with it

File: pkg/resource/github/deserializer/github_membership_deserializer.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	return github.GithubMembershipResourceType
+}
+
+func (s GithubMembershipDeserializer) Deserialize(rawResourceList []cty.Value) ([]resource.Resource, error) {
+	resources := make([]resource.Resource, 0)
+	for _, raw := range rawResourceList {
+		raw := raw
+		res, err := decodeMembership(&raw)
+		if err != nil {
+			return nil, errors.Wrapf(err, "error when deserializing github_team %+v : %+v", raw, err)
```
[the remark this answers is not in the document]
Comment (eliecharra) [reply pkg/resource/github/deserializer/github_membership_deserializer.go]: Nice catch, fixed

**Review by wbeuil — APPROVED** [review]

**Pull request merged**